### ISSUE TITLE
DHS client permissions error - newly added surveys

### ISSUE THREAD

**Issue by kaiserso** [OJWatson/rdhs#29]
Hi,

The following scenario does not work (v. 0.3.0 for package and client)

1. download a bunch of surveys via rdhs
2. request access to 1+ additional survey countries in the existing project from DHS, they grant
3. try to request the new datasets via rdhs gives message "Please request permission for these datasets from the DHS website to be able to download them" for the newly-added datasets

Note if I log in from a separate computer the new datasets are accessible via rdhs.

It appears that rdhs is caching permissions rather than verifying them each time it is instantiated? Note the original system continues to show this message after pulling the latest version and reinstalling rdhs, though I have not attempted manually manipulating the cache.

best,
Peter

**Comment by OJWatson** [comment]
Hey, 

Thanks for flagging this up. This will happen as your previously available datasets have been cached. There is an option to flag this but it's not obvious, and it's not accessible directly using any of the `get_datasets` functions so it's very unclear. I'll add this as an argument to these functions in the next patch (should be today)

You can fix this for now with the following:

```
## if you are still using the client to download datasets
avs <- cli$available_datasets(clear_cache_first = TRUE)
dats <- cli$get_datasets(...)

## with v.0.3.0 onwards
avs <- get_available_datasets(clear_cache_first = TRUE)
dats <- get_datasets(...)

```

**Comment by jeffeaton** [comment]
Can we change `clear_cache_first` to `clear_cache`?  (Lazy user request…)

**Comment by OJWatson** [comment]
...you are so lazy. Yeah sure

**Comment by OJWatson** [comment]
fixed in #30
